sunpy/visualization/draw.py

```diff
@@ -0,0 +1,106 @@
+"""
+This module provides functions that draw on Map plots.
```

Comment:
```suggestion
This module provides functions that draw on WCSAxis.
```
I think this is more correct?